.github/actions/create-project-issue-v1/src/run.test.ts

```diff
@@ -305,4 +322,68 @@ describe('run', () => {
       assert.isTrue(setFailed.calledWith('Column foobar not found'))
     })
   })
+
+  describe('given no `project_number` input', () => {
+    it('defaults to 66', async () => {
+      generateInputs()
+
+      await createAndMoveIssueStub()
+
+      const core = {
+        getInput,
+        setFailed,
+        setOutput,
+        info
+      }
+
+      await run(core as unknown as Core, github as unknown as GitHub)
+
+      // commands that require the project number
+      const [projectBoarArgs, issueAddedArgs, fieldListArgs] =
```

Comment:
This typo is everywhere so will only mark it here, but you'll need to fix it in ever file this has been copy pasted

```suggestion
      const [projectBoardArgs, issueAddedArgs, fieldListArgs] =
```